idlookup/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .idlookup import Idlookup
+
+__red_end_user_data_statement__ = "Channels that idlookup is enabled are stored persistantly."
+
+
+def setup(bot):
+    bot.add_cog(Idlookup(bot))
```

Comment:
Run an autoformatter in both `.py` files if you can find one in vscode, if not nbd we can do it later